lib/luaT/luaT.c

```diff
@@ -471,16 +471,15 @@ void luaT_registeratname(lua_State *L, const struct luaL_Reg *methods, const cha
 }
 
 
-/* utility functions */
-const char *luaT_classrootname(const char *tname)
+const char* luaT_classrootname(const char *tname)
```

Comment:
I'd agree that returning `a.b` makes sense. My one concern there is that `a.b` isn't directly accessible from C (i.e. you can't call `_G['a.b']`).